### PR TITLE
DevKit updates for master branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -45,6 +45,8 @@ matrix:
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: '7.3'
       env: SYMFONY=4.4.*
+    - php: '7.3'
+      env: SYMFONY=5.0.*
```
Comment (greg0ire) [inline .travis.yml]: :thinking: :thinking: :thinking: why?

Comment (OskarStark) [reply .travis.yml]: Should add this to allow failures?

Comment (core23) [reply .travis.yml]: We have no support for symfony 5 for now, to this will always fail

Comment (OskarStark) [reply .travis.yml]: shall we remove this on dev-kit?

Comment (franmomu) [reply .travis.yml]: > shall we remove this on dev-kit?

Let's remove it and add it again when symfony 5 is supported (when SonataIntlBundle is removed), I'll make a PR now.

**Pull request merged**